discord/message.py

```diff
@@ -1991,6 +1993,9 @@ def to_message_reference_dict(self) -> MessageReferencePayload:
 
         return data
 
+    def to_dict(self) -> MessagePayload:
```

Comment:
Could you also implement the `__dict__` dunder method then ?

Reply:
``__dict__`` is not a dunder method, it is a mapping that contains all the attributes of a class if ``__slots__`` is not defined. Doing ``dict(object)`` is the same as doing ``dict((k, v) for k, v in object.__iter__())``.

Reply:
yeah mb @tibue99 can you revert that?

Reply:
sure